reference/orders.v2.oas2.yml

```diff
@@ -4435,6 +4435,18 @@ components:
           description: The transactional currency ID. A read-only value. Do not pass in a POST or PUT.
           example: 1
           type: integer
+        default_currency_code:     
+          description: The currency code of the store's default currency; returned on orders when Multi-Currency is enabled on the store.
+          example: EUR
+          type: string
+        store_default_currency_code: 
+         description: The currency code of the store's default currency; returned on orders when Multi-Currency is enabled on the store.
+         example: USD
+         type: string
+        store_default_to_transactional_exchange_rate:
+          description: The exchane rate between the store's default currency and the transactional currency used in the order; returned on orders when Multi-Currency is enabled on the store.
```

Comment:
The currency code of the store's default currency; returned on orders when Multi-Currency is enabled on the store.
Will b e `1` when the store default currency is the same as the transactional currency.